src/main/java/org/schemaspy/model/Catalog.java

```diff
@@ -19,25 +19,22 @@
 package org.schemaspy.model;
 
 import java.util.Map;
+import java.util.Objects;
 
 import org.schemaspy.util.CaseInsensitiveMap;
 
 public final class Catalog implements Comparable<Catalog>{
     private String name;
     private String comment = null;
-    private Map<String, Schema> schemas = new CaseInsensitiveMap<Schema>();
-    
+
 	public Catalog(String name) {
 		super();
-		 if (name == null)
-	            throw new IllegalArgumentException("Catalog name can't be null");
+		Objects.requireNonNull(name);
 		this.name = name;
 	}
 	
 	public Catalog(String name, String comment) {
-		super();
-		 if (name == null)
-	            throw new IllegalArgumentException("Catalog name can't be null");
+		this(name);
 		this.name = name;
```

Comment:
As the one-argument-constructor is called here (`this(name)`) there is no need to assign the value to the field `name` again. So line 38 can be removed.